src/Command/Humbug.php

```diff
@@ -176,7 +176,7 @@ protected function prepareFinder($directories, $excludes, array $names = null, $
             }
         }
 
-        if (isset($paths)) {
+        if (!is_null($paths) && count($paths) > 0) {
```

Comment:
what was the problematic value? I though it could only be an `array` or `null`

Reply:
When --path option is not present, the default value is an empty array. Because of that "return in_array" is always false.